docs/build_configure_run.md

```diff
@@ -0,0 +1,292 @@
+# Get Started
+
+The `llama` CLI tool helps you setup and use the Llama toolchain & agentic systems. It should be available on your path after installing the `llama-toolchain` package.
+
+This guides allows you to quickly get started with building and running a Llama Stack server in < 5 minutes!
+
+### TL;DR
+Let's imagine you are working with a 8B-Instruct model. We will name our build `8b-instruct` to help us remember the config.
+
+**llama stack build**
```

Comment:
hm. the user is not going to know what "local" means. Didn't we decide that we will just make a bunch of these YAML files and then have the user only specify the YAML file as the only way to do `llama stack build`? having this ease-of-use mode might become confusing because the user now needs to know about these types.